tests/format/yaml/flow-mapping/jsfmt.spec.js

```diff
@@ -1,3 +1,5 @@
 run_spec(__dirname, ["yaml"]);
 run_spec(__dirname, ["yaml"], { tabWidth: 4 });
-run_spec(__dirname, ["yaml"], { yamlBracketSpacing: false });
+// [prettierx] support --no-yaml-bracket-spacing option
+// with bogus option to improve consistency of snapshot with Prettier 2.3.2
+run_spec(__dirname, ["yaml"], { bogus: null, yamlBracketSpacing: false });
```

Comment:
- [ ] use `bogus: true` or `bogus1: true` for improved consistency with other prettierX test updates
- [ ] consider moving this case to the middle of `tests/format/yaml/flow-mapping/jsfmt.spec.js` for more consistency between `run_spec` order and order of test snapshots